### ISSUE TITLE
Dropdown closes for controlled input when time is picked with arrows

### ISSUE THREAD

**Issue by andriy-kudrya-eleks** [YouCanBookMe/react-datetime#90]
react-datetime version - 2.2.1
react version - 15.1.0

Steps to reproduce:
- use controlled value (sample below)
- open time selector
- use up/down buttons to adjust time

Result:
- dropdown is closed after each click

Sample comonent to reproduce bug:
```JavaScript
class MomentInputDemo extends React.Component {
    constructor() {
        super()
        this.state = {time: ''}
    }

    render() {
        return '',
            <ReactDateTime
                value={this.state.time}
                onChange={time => this.setState({time})}
            />
    }
}
```

**Comment by jaraquistain** [comment]
I'm experiencing this same issue

**Comment by andriy-kudrya-eleks** [comment]
Seems to be fixed

**Comment by dbraynard** [comment]
I am also seeing this behavior. Are they any workarounds?

react: 15.4.2
react-datetime: 2.8.7

**Comment by dbraynard** [comment]
FYI, I switched to 2.8.6 and the arrows work fine. It looks like an issue in 2.8.7.

**Comment by aju** [comment]
I have issue with closing popup as well with version 2.8.7 - 2.8.6 works fine!

**Comment by arqex** [comment]
Yes, seems that yesterday version that updates react-onclickoutside also creates new issues. We are working on it, until there is a fix please block your version to 2.8.6.
